demo/index.html

```diff
@@ -66,6 +66,7 @@ <h3>auro-hyperlink examples</h3>
       <demo-snippet>
         <template>
             <p>See more about <auro-hyperlink href="http://www.alaskaair.com">Alaska Airlines</auro-hyperlink> in this link.</p>
+            <p>See more about <auro-hyperlink target="_blank" href="//www.apple.com" noexit>Apple Computers</auro-hyperlink> in this link.</p>
```

Comment:
It seems weird having this be an example of noexit, when this is a link that _will_ exit the domain. Should this be a link to an Alaska page instead?

Reply:
This is the internal demo that is not published to the docsite. Old repo, pay it no mind. Lazy coder.

Reply:
But this raises an interesting question. My 1st approach to this was to make this a manual boolean. Should the solution really be to be more automated based on URL? 

e.g. anything other than `alaskaair` will get the new old icon. 

I like that.

Reply:
I think that has promise, as long as we catch all the edge cases
- desktop
- MOW
- relative links
- does this mean it would behave differently in local development when everything is localhost?

Reply:
![han solo](https://media.tenor.com/images/29bed18465fcb141f3930ebb3023d286/raw)

Reply:
https://github.com/AlaskaAirlines/auro-hyperlink/pull/64/commits/7c8393298318d772e83698b5869da81389167a7b